platform/android/java/README.md

```diff
@@ -0,0 +1,47 @@
+# Third party libraries
+
+
+## Google's vending library
+
+- Upstream: https://github.com/google/play-licensing/tree/master/lvl_library/src/main/java/com/google/android/vending
+- Version: git (eb57657, 2018) with modifications
+- License: Apache 2.0
+
+Overwrite all files under `com/google/android/vending`
+
+### Modify some files to avoid compile error and lint warning
+
+#### com/google/android/vending/licensing/util/Base64.java
+<pre><code>
```

Comment:
You can use
<pre>
```
</pre>
like on GitHub I think. Aren't `pre` + `code` redundant too?

Reply:
Maybe even
<pre>
```diff
...
```
</pre>
to get proper syntax highlighting, at least on GitHub.

Reply:
I will update it soon.